src/main/scala/view/util/scalaQuestSwingComponents/SqSwingFileChooser.scala

```diff
@@ -0,0 +1,38 @@
+package view.util.scalaQuestSwingComponents
+
+import controller.util.Resources.ResourceName
+
+import java.io.File
+import javax.swing.filechooser.FileNameExtensionFilter
+import javax.swing.{JComponent, JFileChooser}
+
+/**
+ * Utility object used to get and show elements in a FileChooser.
+ */
+object SqSwingFileChooser {
+
+  def showFileSave(title: String, onSave: String => Unit, selectedFileName: String, parent: JComponent): Unit = {
```

Comment:
This def should not be here.
Just leave the case class and remove the object. The method will go where needed.

Reply:
@PieroSanchi i added a comment.